TopMovies/View/MovieCategoryTableViewCell.swift

```diff
@@ -13,13 +13,10 @@ struct MovieCategoryProps {
   let movies: [MovieCollectionProps]
 }
 extension MovieCategoryProps {
-  init?(categoryNameText: String?, movies: [MovieCollectionProps]?) {
-    guard
-      let categoryNameText = categoryNameText,
-      let movies = movies
-    else { return nil }
+  init?(categoryNameText: String?, movies: [MovieCollectionProps?]) {
+    guard let categoryNameText = categoryNameText else { return nil }
     self.categoryNameText = categoryNameText
-    self.movies = movies
+    self.movies = movies.compactMap{ $0 }
```

Comment:
What is `compactMap { $0 }` for?

Reply:
MovieCollectionProps.Type has <init?(movie: Movie?)>, so it can be nil, compactMap helps with unwraping in this case